xml/System/String.xml

```diff
@@ -9936,9 +9930,9 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
 ## Examples
  The following example demonstrates the <xref:System.String.Length%2A> property.
 
- [!code-cpp[string.length#1](~/samples/snippets/cpp/VS_Snippets_CLR/string.length/CPP/length.cpp#1)]
- [!code-csharp-interactive[string.length#1](~/samples/snippets/csharp/VS_Snippets_CLR/string.length/CS/length.cs#1)]
- [!code-vb[string.length#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/string.length/VB/length.vb#1)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/string.length/CPP/length.cpp" id="Snippet1":::
```

Comment:
weird... this change made the C# and C++ samples disappear now 🤔 
Maybe we should roll back the change for these 3 lines